docs/src/main/asciidoc/spring-cloud-sleuth.adoc

```diff
@@ -60,6 +57,123 @@ NOTE: Remember that tags are only collected and exported if there is a
 danger of accidentally collecting too much data without configuring
 something).
 
+NOTE: Currently the instrumentation in Spring Cloud Sleuth is eager - it means that
+we're actively trying to pass the tracing context between threads.
+This approach may change in the future towards being lazy on this matter.
+
+== Span lifecycle
+
+You can do the following operations on the Span by means of *Tracer* interface:
+- *start* - when you start a span its name is assigned and start timestamp is recorded.
+- *close* - the span gets *stopped* (the end time of the span is recorded) and if
+the span is *exportable* then it will be eligible for collection to Zipkin.
+The span is also removed from the current thread.
+- *continue* - a new instance of span will be created whereas it will be a copy of the
```

Comment:
this sentence is confusing. and the example is also confusing as it doesn't say why I wouldn't just use the reference of spanToContinue